jwst/extract_2d/grisms.py

```diff
@@ -192,7 +194,7 @@ def extract_tso_object(input_model,
             output_model.dq = ext_dq
             output_model.meta.wcs = subwcs
             output_model.meta.wcs.bounding_box = util.wcs_bbox_from_shape(ext_data.shape)
-            output_model.meta.wcs.crpix2 = 34  # update for the move, vals are the same
+            output_model.meta.wcs.siaf_yref_sci = 34  # update for the move, vals are the same
```

Comment:
Above siaf_yref_sci is part of the meta.wcs**info** attribute, but here it's part of meta.wcs

Reply:
Hm, good point. I'll investigate.

Reply:
fixed - appears to be an old bug